pom.xml

```diff
@@ -46,7 +46,7 @@
         <embedded-artemis.version>1.2.0</embedded-artemis.version>
         <jboss-ejb3-ext-api.version>2.2.0.Final</jboss-ejb3-ext-api.version>
         <test-utils.version>1.22.0-M1</test-utils.version>
-        <utilities.version>1.16.4</utilities.version>
+        <utilities.version>1.17.0-M3</utilities.version>
```

Comment:
maybe we should have released utilities as a full release rather than Milestone release...we should sort that out later

Reply:
fixed